sources/ElkArte/AdminController/Maintenance.php

```diff
@@ -1051,7 +1051,7 @@ public function action_backup_display()
 		checkSession('post');
 
 		// Validate access
-		if (defined('I_KNOW_IT_MAY_BE_UNSAFE') === false && $this->_validate_access() === false)
+		if (!defined('I_KNOW_IT_MAY_BE_UNSAFE') && $this->_validate_access() === false)
```

Comment:
Just for consistency's sake, _recently_ I've been using ```=== false``` in this type of comparisons mostly because from time to time the ```!``` seems to me less easy to catch.
I don't really mind one way or the other, so should we always use the ```!```?

Reply:
I don't think it matters that much, but since when we are already dealing with a boolean value why do another logical compare to get a boolean value again?  I think using the ! with a well named variable makes for cleaner code.